api/mod/src/routers/users.py

```diff
@@ -0,0 +1,48 @@
+"""Routes for users"""
```

Comment:
From what I understood, `fastapi_restful` should help us not having to write these bloat CRUD functions ourselves

Reply:
what exactly do you mean by that? I think I understand the idea (using somewhat predefined CRUD operations from fastapi_restful instead of defining all of them ourselves?) but didn't really find any specific information on how to do that.

The closest I found was that: https://fastapi-restful.netlify.app/user-guide/class-resource/

Reply:
Hm, good question.
From Ruby on Rails, I know the concept of just defining a class that knows how to construct, store, edit and so on itself, and then the API routes for creating a new instance, getting an instance, editing an instance, listing all instances etc got generated automatically based on this.
I somehow thought fastapi-restful works similarly, but does not really look so, does it?

Reply:
Class Resource is most similar to what I meant, but it seems not to ease things as much as I would have expected it to :/

Reply:
We should just drop fastapi_restful

Reply:
Yes I thought so too but I didn't really find any good examples how this Class Resource is actually implemented. And as we are quite familiar with the usual routing style we used so far I also think we can just stick with that.